index.html

```diff
@@ -0,0 +1,150 @@
+                </ul>
+            </div>
+
+            <div class="left-box three-columns align-center">
+                    <img class="picture-sm" src="icons/anica.png" alt="anica">
+                    <p class="text-name">Anica Rahman<br>
+                    @Solo Ann</p>
+                    <img class="more-icon" src="icons/more.png" alt="more">
+                </div>
+            </div>
+
+            <!--MIDDLE SECTION LAYOUT-->
+        <div class="tweet-feed">
+            <div class="middle-box">
+                <h2>Home</h2>
+            </div>
+            <div class="middle-box">
+                <div class="post two-columns">
+                    <img class="picture-sm" src="icons/anica.png" alt="anica">
+                    <p class="text">What's happening?</p>
+                </div>
+                <div class="post seven-columns">
+                    <div><img src="icons/image.png" alt="image"></div>
+                    <div><img src="icons/gif.png" alt="gif"></div>
+                    <div><img src="icons/statistics.png" alt="statistics"></div>
+                    <div><img src="icons/happy.png" alt="happy"></div>
+                    <div><img src="icons/calendar.png" alt="calendar"></div>
+                    <div class="space"></div>
+                    <button>Tweet</button>
+                </div>
+            </div>
+            <div class="middle-box">
+                <div class="post two-columns">
+                    <img class="picture-sm" src="icons/anica.png" alt="anica">
+                    <p><strong>Anica</strong> - Here is my Twitter clone. What do you think?</p>
+                </div>
+                <div class="post">
+                    <ul class="reaction-section grid-auto-column">
+                        <li> <img src="icons/heart.png" alt="heart"></li>
+                        <li><img src="icons/comment.png" alt="comment"></li>
+                        <li><img src="icons/share.png" alt="share"></li>
+                        <li><img src="icons/repeat.png" alt="repeat"></li>
+                    </ul>
+                </div>
+            </div>
+            <div class="middle-box">
+                <div class="post two-columns">
+                    <img class="bear-image" src="icons/bear.png" alt="bear">
+                    <p><strong>Elena</strong> - Well done! I'm proud of you.</p>
+                </div>
+                <div class="post two-columns">
+                    <img class="inter-image" src="icons/inter.png" alt="inter">
+                    <p><strong>Nabil</strong> - Is that it? You need to work on it a bit more.</p>
+                </div>
+                <div class="post two-columns">
+                    <img class="khaled-image" src="icons/khaledhosseini.jpeg" alt="inter">
+                    <p><strong>Khaled Hosseini</strong> - Awesome! This is better than the original version.</p>
+                </div>
```

Comment:
You could make all the images into one class (e.g "post-image") since all work the same with the same sizes